src/table/components/TableHeadWrapper.jsx

```diff
@@ -18,16 +19,21 @@ function TableHeadWrapper({
   translator,
   selectionsAPI,
   keyboard,
-  children,
 }) {
   const { columns, paginationNeeded } = tableData;
+  const setHeadRowHeight = useContextSelector(TableContext, (value) => value.setHeadRowHeight);
   const isFocusInHead = useContextSelector(TableContext, (value) => value.focusedCellCoord[0] === 0);
   const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
   const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme]);
+  const headRowRef = useRef();
+
+  useEffect(() => {
+    setHeadRowHeight(headRowRef.current.clientHeight);
+  }, [headRowRef.current, headerStyle.fontSize]);
```

Comment:
check this:

https://user-images.githubusercontent.com/4471489/177321037-a95a2949-3638-4ea7-adaa-f3f1e1c56d4a.mov

Reply:
I wonder how you reproduce it because I can't. Have you set a specific fontSize? You saw the table in my screen, right? I changed the screen size and fontSize but never get the same locally